lte/gateway/python/magma/subscriberdb/tests/client_tests.py

```diff
@@ -272,3 +280,38 @@ def test_detach_deleted_subscribers(self, s6a_service_mock_stub):
         mock.DeleteSubscriber.future.assert_called_once_with(
             DeleteSubscriberRequest(imsi_list=["101", "303"]),
         )
+
+    @ unittest.mock.patch(
+        'magma.common.service_registry.ServiceRegistry.get_rpc_channel',
+    )
```

Comment:
What does this do

Reply:
I think this creates a local mock for the cloud service registry (hence all test cases that make rpc calls to the cloud use this)